include/grpc++/impl/call.h

```diff
@@ -90,10 +93,12 @@ class CallOpBuffer : public CompletionQueueTag {
   grpc_metadata_array recv_initial_metadata_arr_;
   // Send message
   const grpc::protobuf::Message *send_message_;
-  grpc_byte_buffer *send_message_buf_;
+  const ByteBuffer *send_message_buffer_;
+  grpc_byte_buffer *send_buf_;
```

Comment:
I can do that too, but merging the two will require one more field to indicate whether a proto message was passed in or a ByteBuffer was passed in. The ByteBuffer* is more symmetric to the message*.